zerver/tests/test_email_change.py

```diff
@@ -103,7 +105,8 @@ def test_end_to_end_flow(self) -> None:
         from_email = email_message.from_email
         self.assertIn('We received a request to change the email', body)
         self.assertIn('Zulip Account Security', from_email)
-        self.assertIn(FromAddress.NOREPLY, email_message.from_email)
+        tokenized_no_reply_email = parseaddr(email_message.from_email)[1]
+        assert(re.search(self.TOKENIZED_NOREPLY_REGEX, tokenized_no_reply_email))
```

Comment:
I think we prefer `self.assertTrue` to `assert` where we're making assertions about whether the real code works.  I like `assert()` for cases where we're asserting some invariant about the test itself, which doesn't seem to be the case here.